src/app/filter_utils.ts

```diff
@@ -143,9 +142,9 @@ const MILLISECONDS_PER_DAY = 1000 * 60 * 60 * 24;
 // differently. We're also interpreting weeks and months relative to today, 
 // but these could also be determined based on where we are in the month.
 // Added an optional filterType parameter to allow for the option of using the current time as the start time instead of midnight.
-export function buildDateFilterForNDays(now: Date, days: number,filterType: DayFilterType = DayFilterType.MIDNIGHT): DateFilter {
+export function buildDateFilterForNDays(now: Date, days: number,fromFilter:DayFilterType=DayFilterType.MIDNIGHT): DateFilter {
```

Comment:
```suggestion
export function buildDateFilterForNDays(now: Date, days: number, fromFilter: DayFilterType = DayFilterType.MIDNIGHT): DateFilter {
```
spacing... Doesn't the project include any linting/formatting for ts files?

Reply:
No linter

Reply:
Don't any of these lint/format the TS code @kelvinkipruto: https://github.com/CodeForAfrica/harassment-manager/blob/714df80b4a9dfd6243ce55eb57470caf35d7a384/package.json#L13-L14